ironfish/src/testUtilities/fixtures/transactions.ts

```diff
@@ -99,6 +100,48 @@ export async function useTxFixture(
   })
 }
 
+export async function useUnsignedTxFixture(
+  wallet: Wallet,
+  from: Account,
+  to: Account,
+  generate?: FixtureGenerate<UnsignedTransaction>,
+  fee?: bigint,
+  expiration?: number,
+): Promise<UnsignedTransaction> {
+  generate =
+    generate ||
+    (async () => {
+      const raw = await wallet.createTransaction({
+        account: from,
+        outputs: [
+          {
+            publicAddress: to.publicAddress,
+            amount: BigInt(1),
+            memo: '',
+            assetId: Asset.nativeId(),
+          },
+        ],
+        fee: fee ?? 0n,
+        expiration: expiration ?? 0,
+        expirationDelta: 0,
+      })
+      Assert.isNotNull(from.spendingKey)
+      const key = generateKeyFromPrivateKey(from.spendingKey)
```

Comment:
Ugly requirement because we don't have `proof_generation_key` exposed directly in our accounts. We will probably update that soon.